sklearn/utils/_testing.py

```diff
@@ -688,7 +688,12 @@ def _get_diff_msg(docstrings_grouped):
 
 
 def _check_consistency_items(
-    items_docs, type_or_desc, section, n_objects, descr_regex_pattern=""
+    items_docs,
+    type_or_desc,
```

Comment:
Can we document that the possible values for `type_or_desc` is "type specification" and "description"?

Reply:
Thanks this function was definitely worth a parameter docstring.